### ISSUE TITLE
Pull changes from codespell dictionary?

### ISSUE THREAD

**Issue by luzpaz** [orbitcowboy/codespell_dictionary#13]
Is this not happening anymore?

**Comment by orbitcowboy** [comment]
Yes, you can do this by executing https://github.com/orbitcowboy/codespell_dictionary/blob/master/importFromCodespell.sh 
I am doing this in regular intervals. But feel free to do it yourself :-)

**Comment by orbitcowboy** [comment]
Could you give it a try an import data from codespell? I was working on it recently

https://github.com/orbitcowboy/codespell_dictionary/commit/da8522902a1c38a801cb508c592587074c079c1a

**Comment by luzpaz** [comment]
yea.. will try very soon.

**Comment by luzpaz** [comment]
Just ran it, it tries add a ton of duplicates

Here are the different usecases:

* codespell tries to overwrite a valid orbitcowboy single string result  
* codespell tries to overwrite a valid orbitcowboy mutli-string result:  

  ```diff
  +absoluve->absolute
  absoluve->absolve, absolute,
  ```

* codespell is correct and has just re-arranged results 

  ```diff
   actuall->actual, actually,
  +actuall->actually, actual,
  ```

**Comment by orbitcowboy** [comment]
Yes, thats indeed a problem. We have to improve the script in order to reliably import data from codespell.

**Comment by luzpaz** [comment]
I bet we could grab the tests that are done in the TravisCI and customize those scripts. Though coding is not my jam, so much.
